concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-132	132	chique	chic			8.1	4.2	4.25	5.25	5.1	5.55	4.75	6	1	18036	2	374	41	2.55719685120929	0.000366040210501
+Verheyen-2019-1000-133	133	cirkelvormig	circular	CIRCULAR (ROUND IN TWO DIMENSIONS)	2302	6.55	3.15	6.2	3.55	3.55	6.65	3.65	12	4	61339	0	116	2	0.038361042613291	6.93140873170059E-06
+Verheyen-2019-1000-134	134	commercieel	commercial			11	4.45	3.65	4.7	4.6	4.75	3.85	11	3	107111	0	787	198	0.411289096959908	7.97339796920788E-05
+Verheyen-2019-1000-135	135	communistisch	communist			12.95	4.05263157894737	5.7	4.7	3.3	4.95	3.15	13	4	89696	0	1118	189	0.043770754381441	6.26755550105885E-06
+Verheyen-2019-1000-136	136	compleet	complete			8.55	4.45	4.65	5.05	5.75	4.35	4.9	8	2	40177	0	1368	1969	0.588710195007943	5.36777403893903E-05
+Verheyen-2019-1000-137	137	conventioneel	conventional			13.5263157894737	3.6	3.5	3.6875	2.5	2.78947368421053	4.15789473684211	13	4	131084	0	456	71	0.026406325619987	6.98998401675722E-06
+Verheyen-2019-1000-138	138	correct	correct	CORRECT (RIGHT)	1725	6.75	3.8	4.85	4.75	4.65	4.3	5.35	7	2	32786	0	543	751	2.90520469279606	0.00026425914435
+Verheyen-2019-1000-139	139	corrupt	putrid			10.95	4.5	5.25	4.35	3.25	4.5	2.35	7	2	18084	0	177	468	0.357126376893628	7.22070555623026E-05
+Verheyen-2019-1000-140	140	creatief	creative			8.2	5.1	4.75	5.25	4.95	3.7	5.3	8	3	55958	0	947	452	3.20122982838434	0.000672265292412
+Verheyen-2019-1000-141	141	cultureel	cultural			10.6	4.45	4.7	4.1	4.75	3.8	5.1	9	3	56022	0	2469	221	0.16658510471666	3.34497419498359E-05
+Verheyen-2019-1000-142	142	dagelijks	daily			7.25	4.45	6	3.7	5.5	4.35	4.2	9	3	66933	0	3978	765	1.93714470417292	0.000339567435838
+Verheyen-2019-1000-143	143	dankbaar	grateful			7.45	4.4	5.1	4.25	5.25	4.6	5.7	8	2	50211	1	1434	1641	0.619558628376839	0.000104176144473
+Verheyen-2019-1000-144	144	dapper	brave	BRAVE	3	6.65	5.65	4.85	5.75	3.75	4.85	5.75	6	2	54619	7	695	1395	1.03884899739172	0.000143200301051
+Verheyen-2019-1000-145	145	decadent	decadent			13.55	4.44444444444445	3.94117647058824	5.16666666666667	2.55	4.95	2.88888888888889	8	3	77278	0	168	57	0.302752510610197	0.000143008304283
+Verheyen-2019-1000-146	146	definitief	final			10	4.35	5.35	5.75	4.8	4.3	3.8	10	4	80334	0	1994	379	0.906134278405817	0.000145819917966
+Verheyen-2019-1000-147	147	deftig	distinguished			7.36842105263158	3.7	4.45	4.6	4.7	6	4.85	6	2	35143	2	647	153	2.45425437024628	0.000384650880237
+Verheyen-2019-1000-148	148	democratisch	democratic			11.6	4.55	5.25	4.25	4.35	4.7	5.05	12	4	86368	0	1553	162	0.095882771835186	2.33520136425744E-05
+Verheyen-2019-1000-149	149	depressief	depressed			9.85	2.4	4.55	4.2	5.1	4.45	1.5	10	3	67938	2	246	518	1.71757859227863	0.000262098367168
+Verheyen-2019-1000-150	150	derde	three			5.05	3.6	5.31578947368421	3.25	4.7	4.5	3.4	5	2	71938	1	0	1	0.471476297335574	0.000108351260625
+Verheyen-2019-1000-151	151	deugdzaam	good	GOOD	1035	11.1	3.65	4.05263157894737	4.15	3.2	3.8	5.55	9	2	39406	0	85	53	0.074234850877636	3.6554232057837E-05
```

Comment:
deugdzaam is virtuous, morally good (cf. German tugendhaft), concept GOOD might be too broad?